src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  margin: auto;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background-color: $white;
+  justify-content: center;
+
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    text-align: left;
+    padding-top: 40px;
+    color: $main-color;
+  }
+
+  &__product-code {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    padding-top: 4px;
+    color: $secondary-color;
+  }
+
+  &__stars-rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 16px;
+    line-height: 18px;
+  }
+
+  &__cost {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding-top: 24px;
+    padding-bottom: 16px;
+  }
+
+  &__cost-title {
+    font-weight: 400;
+    text-align: left;
+  }
+
+  &__cost-price {
+    font-size: 16px;
+    font-weight: 700;
+    text-align: right;
+  }
+
+  &__buttom {
+    box-sizing: border-box;
+    display: flex;
+    background-color: $blue-accent;
+    width: 166px;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. Just use padding